[2024-01-17T19:32:10.990-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:30:50.589390+00:00 [queued]>
[2024-01-17T19:32:10.993-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:30:50.589390+00:00 [queued]>
[2024-01-17T19:32:10.993-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T19:32:11.001-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-17 22:30:50.589390+00:00
[2024-01-17T19:32:11.006-0300] {standard_task_runner.py:57} INFO - Started process 87020 to run task
[2024-01-17T19:32:11.008-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_coleta_teste', 'df', 'scheduled__2024-01-17T22:30:50.589390+00:00', '--job-id', '2545', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp6ij7qz4f']
[2024-01-17T19:32:11.008-0300] {standard_task_runner.py:85} INFO - Job 2545: Subtask df
[2024-01-17T19:32:11.035-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_coleta_teste.df scheduled__2024-01-17T22:30:50.589390+00:00 [running]> on host desktop
[2024-01-17T19:32:11.102-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_coleta_teste' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T22:30:50.589390+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T22:30:50.589390+00:00'
[2024-01-17T19:32:13.731-0300] {logging_mixin.py:154} INFO - Called | 1705530733.7288225 | df
[2024-01-17T19:32:13.738-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-17T19:32:13.739-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T19:32:13.742-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_coleta_teste, task_id=df, execution_date=20240117T223050, start_date=20240117T223210, end_date=20240117T223213
[2024-01-17T19:32:13.824-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T19:32:13.832-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
